src/components/PublicHeader.tsx

```diff
@@ -0,0 +1,47 @@
+interface PublicHeaderProps {
+    userData: {
+        avatar: string;
+        first_name: string;
+        id: number;
+        last_name: string;
+    }
+}
+
+const PublicHeader = ({ userData } : PublicHeaderProps) : JSX.Element => {
+    return (
+        <header>
+            <div style={{
+                alignItems: 'center',
+                display: 'flex',
+                justifyContent: 'space-between',
+                padding: '1rem'
+            }}>
+                <img
+                    data-test='zetkin-logotype'
+                    alt='Zetkin logo'
+                    src='/logo-zetkin.png'
+                    height={ 33 }
+                    width={ 40 }
+                />
+                { !userData ? <button data-test='login-button'>Login</button> : null }
+                { userData ? 
+                    <div style={{ alignItems: 'center', display: 'flex' }}>
+                        <p>{ userData.first_name } { userData.last_name }</p>
+                        <img 
+                            data-test='user-avatar'
+                            alt='User avatar' 
+                            src={ userData.avatar }
```

Comment:
Construct avatar URL here instead of relying on an `avatar` attribute which will not exist on user objects coming from the Zetkin API.

The path to a user's avatar is https://api.zetk.in/v1/users/{user_id}/avatar, e.g. https://api.zetk.in/v1/users/2/avatar for myself.

Remember that you have an API proxy set up, so you don't have to retrieve data from `https://api.zetk.in/v1/…` (which means you're hard-coding the live Zetkin URL into the code). The same path will be available at `/api/…` thanks to the API proxy you set up.